reframe/core/schedulers/slurm.py

```diff
@@ -431,10 +429,11 @@ def __init__(self, node_descr):
             raise JobError('could not extract NodeName from node description')
 
         self._partitions = self._extract_attribute(
-            'Partitions', node_descr, sep=',')
+            'Partitions', node_descr, sep=',') or set()
```

Comment:
Yes, that is the purpose of the `set()`. An empty set in not always valid since for `_name` we want to through an exception if it cannot be extracted.

Reply:
Ok, it makes sense.